### PR TITLE
calling fcm instead of gcm

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

**Comment by codiebeulaine** [comment]
@miltontony ready for review

**Comment by miltontony** [comment]
👍 🍰 once travis!

File: mentorship/mentor/utils.py [hunk 1]
```diff
@@ -34,25 +34,25 @@ def send_reset_email(to_email, context):
...
-def send_notification_to_gcm(
+def send_notification_to_fcm(
         mentor, title, content, object_id, use_notification=True):
-    try:
-        gcm = GCM(settings.GCM_API_KEY)
-    except AttributeError:
-        return False
 
+    if not hasattr(settings, 'FCM_DJANGO_SETTINGS'):
+        return False
```
Comment (justinvdm) [inline mentorship/mentor/utils.py]: We don't seem to be using this setting. Should we be accessing the auth key off of it?

Comment (codiebeulaine) [inline mentorship/mentor/utils.py]: The Device checks for it in the settings and uses it behind the scenes

**Pull request merged**